apps/wakunode2/app.nim

```diff
@@ -127,6 +127,21 @@ proc init*(T: type App, rng: ref HmacDrbgContext, conf: WakuNodeConf): T =
       quit(QuitFailure)
     else: recordRes.get()
 
+  # Check the ENR sharding info for matching config cluster id
```

Comment:
We may want to figure out some method of configuration that doesn't allow for inconsistencies here (i.e. only configuring cluster ID and shards, deprecating any `--pubsub-topic` configuration)

Reply:
Yes having 2 ways of setting the cluster id is not super ergonomic.